src/cdomains/lval.ml

```diff
@@ -385,22 +385,94 @@ end
 module NormalLatRepr (Idx: IntDomain.Z) =
 struct
   include NormalLat (Idx)
+  module ReprOffset = struct
```

Comment:
This is all just copy-paste from existing code... `Lval.offs` is polymorphic in both offset types, so the existing type can simply be reused. `Lval.OffsetPrintable` is a functor, so it could simply be instantiated with a different (unit) index domain without copying all this code.

Reply:
Yes, good idea! I now created PR #926 that implements that approach. Since we might want to still merge the current PR first (because we already have a benchmark running), I made it a separate PR.